<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/imagedownloader.qrc">
    <normaloff>:/images/github plaatje.png</normaloff>:/images/github plaatje.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="searchFields" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="tagLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tags</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tagField">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="ratingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ratingBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Safe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Safe/Questionable</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Questionable</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Questionable/Explicit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Explicit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="siteName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>31</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Site</string>
    </property>
   </widget>
   <widget class="QComboBox" name="siteBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Konachan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yande</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Danbooru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gelbooru</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/imagedownloader.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
